testhttp/__init__.py

```diff
@@ -5,6 +5,7 @@
 import requests
 import re
 import random
+import collections
 
 version = '0.5.1'
```

Comment:
Feel free to increment the version too. I think I have it in couple of files.

Reply:
Bumped to 0.6.0